nms/app/packages/magmalte/scripts/fuji-upgrade/runs-on-nms.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+###############################################################################
+# Copyright 2020 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###############################################################################
+
+# Script arguments (specifies source DB only):
+# -u: DB username
+# -w: DB password
+# -h: DB host
+# -p: DB port
+# -b: DB name
+# -d: DB SQL dialect
+#
+# EXAMPLE USAGE
+# ./runs-on-nms.sh -h postgres -p 5432 -b nms -d postgres -u root -w password
+
+while getopts ":h:p:b:d:u:w:" opt; do
+  case $opt in
+    h) arg_host="$OPTARG"
+    ;;
+    p) arg_port="$OPTARG"
+    ;;
+    b) arg_db="$OPTARG"
+    ;;
+    d) arg_dialect="$OPTARG"
+    ;;
+    u) arg_username="$OPTARG"
+    ;;
+    w) arg_password="$OPTARG"
+    ;;
+    \?) echo "Invalid option -$OPTARG" >&2
+    ;;
+  esac
+done
+
+cat << EOF
+--------------------------------------------------------------------------------
+              Upgrading @fbcnms/sequelize-models to ^0.1.4
+--------------------------------------------------------------------------------
+EOF
+pushd /usr/src/packages/magmalte
```

Comment:
You can alternatively, create a new directory /usr/src/packages/preupgrade and do 
yarn yarn add -W @fbcnms/sequelize-models@^0.1.4 --modules-folder ./ which will install it in that directory and you can run the script from there.

Reply:
I'll just leave this as is, I think, since the upgrade shouldn't break NMS